plugins/src/main/java/com/qubole/quark/plugins/qubole/QuboleDB.java

```diff
@@ -77,7 +77,8 @@
           .put(Primitive.DOUBLE.primitiveClass.getSimpleName(), Types.DOUBLE)
           .put("date", Types.DATE)
           .put("time", Types.TIMESTAMP)
-          .put("boolean", Types.BOOLEAN).build();
+          .put("boolean", Types.BOOLEAN)
+          .put("real", Types.FLOAT).build();
```

Comment:
real can map to either double or float depending on data source. For redshift though it maps to float. So can you move this to Redshift specific file ?